internal/yamlwriter/yamlwriter.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package yamlwriter
+
+import (
+	"fmt"
+	"io"
+
+	"gopkg.in/yaml.v3"
```

Comment:
any reason to use v3 instead of v2 which we already have?

Reply:
I think we can remove this package entirely. Unless we want a new output format - YAML. The importer will use only one output format, and it is the one from Kubernetes YAML/JSON serializer.

Reply:
I'm ok to remove, I think I created a ticket somewhere to update to yaml.v3 we never get to and we also are in talks to at some point support yaml output but we need to figure out if we'll add yaml tags to the atlas api objects

Reply:
It works without adding any tags, just like with json serialization, the field name will be taken as a tag. Shall I keep it though? Or it is better to remove it?

Reply:
Let's remove this and use v2. We should move everything to v3 if we want to use it and we don't need to do it here.
FYI: I quickly looked at the yaml repo and there is this [yaml.v3 incompatible with yaml.v2 #783](https://github.com/go-yaml/yaml/issues/783) to keep in mind if we plan in the future to upgrade to v3